tests/integration/defs/accuracy/test_llm_api_autodeploy.py

```
@@ -161,6 +161,31 @@ def low_memory_overrides(config,
     return config
 
 
+def reduced_model_kwargs(num_hidden_layers: int,
+                         model_path: str = None) -> dict:
```

Ruff (0.15.9)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)